ui/src/app/utils/user-apps-utils.spec.tsx

```diff
@@ -78,14 +78,25 @@ describe('User Apps Helper functions', () => {
   it('Update User Apps that does not require a subsequent update', async () => {
     const spyListAppsAPI = jest
       .spyOn(appsApi(), 'listAppsInWorkspace')
-      .mockImplementation(
-        () =>
-          new Promise<ListAppsResponse>(() => [
-            { status: RuntimeStatus.RUNNING, appType: AppType.CROMWELL },
```

Comment:
this test wasn't valid because this never actually executed: note how the type for status is wrong here